src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -326,10 +333,19 @@ public void addRow(T item, List<Content> contents) {
             HtmlStyle cellStyle = columnStyles.get(colIndex);
             // Always add content to make sure the cell isn't dropped
             var cell = HtmlTree.DIV(cellStyle).addUnchecked(c.isEmpty() ? Text.EMPTY : c);
+            boolean matchFound = c.isEmpty() || checkFormElements.matcher(c.toString()).find();
             cell.addStyle(rowStyle);
+            if (!matchFound) {
+                cell.put(HtmlAttr.ROLE, "tablist")
+                    .put(HtmlAttr.TABINDEX, "0");
+            }
```

Comment:
This seems to apply to many more places than the title suggests - I don't think this is what we want.

Reply:
These values should be applied to all grid rows with/without tab elements which has only plain text.